prep-javascript-arrays/main.js

```diff
@@ -0,0 +1,10 @@
+const languages = ['html', 'css', 'javascript', 'react', 'node', 'postgres'];
+console.log(languages);
+var firstElement = languages[0];
+console.log('the first element of the array is', firstElement);
+var thirdElement = languages[2];
+console.log('the 3rd element of the array is', thirdElement);
+var length = languages.length;
+console.log('The length of the array is', length);
+var lastElement = languages[5];
```

Comment:
We can calculate the last index by subtracting 1 from the length

```js
var lastElement = languages[length - 1];
```

Reply:
I made the correction and pushed again, thank you!